### PR TITLE
Refactors to fix some issues.

### LINKED ISSUE #153: Bump timeouts in fedora_io method in attacher.rb service concern
To prepare for the big export I'm going to recommend we change the timeout on L232 in the attacher.rb concern 
specifically this line 
`client.timeout(connect: 5, read: 10)` 
the `read` timeout should be bumped to like 300 just to be safe

### LINKED ISSUE #154: File Sets are not being rolled back properly on create
File sets are not rolling back on create. This is causing validation errors during the export script.

### PR DESCRIPTION
**Pull request by bbarberBPL**
-Refactored model scopes

-Refactored attacher service concern to handle createing/updating of blobs safer, Resolves #154

-Used find_or_initialize_by.tap for factory services

-Bumped fedora import timeout in attacher serivce concern, Resolves #153

-Bumped RAILS_VERSION to recent 6.1 stable in travis

-Fixed issue in digital_objects_controller spec not grabbing is_member_of_collection association properyly when setting config.load_defaults 6.1 in internal test apps config/application.rb

-tweaked validations on exemplary_image mappings

### CODE REVIEW

File: lib/curator/services/factory_service.rb [hunk 1]
```diff
@@ -85,7 +85,7 @@ def find_or_create_nomenclature(nomenclature_class:, term_data: {}, authority_co
             Rails.logger.error '==============================================='
             Rails.logger.error e.backtrace.join("\n")
             Rails.logger.error '==============================================='
-            raise e
+            raise
```
Comment (ebenenglish) [inline lib/curator/services/factory_service.rb:88]: intentional to call `raise` with no args?

File: app/services/concerns/curator/filestreams/attacher.rb [hunk 2]
```diff
@@ -94,22 +97,21 @@ def ingest_content?(io = {})
...
-            metadata: attachment_attributes['metadata']
-          ).tap do |blob|
+          ActiveStorage::Blob.find_or_initialize_by(key: attachment_attributes['key']).tap do |blob|
+            blob.filename =  attachment_attributes['file_name']
+            blob.byte_size = attachment_attributes['byte_size']
+            blob.checksum = checksum_to_base64(attachment_attributes['checksum_md5'])
+            blob.content_type = attachment_attributes['content_type']
+            blob.service_name = attachment_attributes['service_name']
+            blob.metadata = attachment_attributes['metadata']
+          end.tap(&:save!).tap do |blob|
```
Comment (ebenenglish) [inline app/services/concerns/curator/filestreams/attacher.rb:107]: Double `tap` here intentional?

**Review by ebenenglish — COMMENTED** [review]
A couple questions, see file-level comments

File: app/services/concerns/curator/filestreams/attacher.rb (hunk 2 above)
Comment (bbarberBPL) [reply app/services/concerns/curator/filestreams/attacher.rb:107]: Per these methods https://github.com/rails/rails/blob/e3e3a97ada3f2b4c0c8d48f941c2ea05c859cdda/activestorage/app/models/active_storage/blob.rb#L97

I basically mimic'd how these methods chain their taps. Alos this prevents the blob from getting uploaded if the `save!` call fails

File: lib/curator/services/factory_service.rb (hunk 1 above)
Comment (bbarberBPL) [reply lib/curator/services/factory_service.rb:88]: This actually re raises the previous error according to this answer 
https://stackoverflow.com/a/23771227

**Pull request merged**